public/assets/css/main.css

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright 2015 UC Berkeley (UCB) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+body {
+  overflow-y: hidden;
```

Comment:
Without it, you can see the iFrame's scrollbar flicker in and out when the iFrame is being resized. Hiding the scrollbar essentially makes it a much nicer experience where the scrollbar flickering is invisible to the user. Data cultures uses this approach as well and it seems to be working well so far.